src/_scss/pages/search/timePeriodFilter.scss

```diff
@@ -0,0 +1,38 @@
+.timePeriodFilter {
+
+    @import './datePicker';
+
+    .fiscalYears {
+        margin: 1.5rem 0 3rem 0;
+    }
+
+    label {
+        font-size: $small-font-size;
+        margin:0;
+    }
+
+    .fiscalYearOption {
+        width:16rem;
+        max-width:160px;
+        margin:0;
+        display:table-cell;
+        height:2.5rem;
+    }
+
+    .fiscalYearOption input {
+        max-height:1.5rem;
+    }
+
+    .fiscalYearOption label::before {
+        height:1.2rem;
+        width:1.2rem;
+    }
+
+    .fiscalYearOption:nth-child(odd){
+        float:left;
```

Comment:
We have Bourbon and Neat. Can we use the grid system instead of floating ourselves? Also why are we floating inside a `display: table-cell`?